cmd/buildah/config.go

```diff
@@ -17,6 +18,10 @@ import (
 
 var (
 	configFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "add-history",
```

Comment:
Lets add an environment variable to turn this on or off by default.

Reply:
Added `$BUILDAH_HISTORY` for it.